Research landing page

mattosborn opened the pull request:
Tidied up from the hackathon and added a new topic map for testing

Topic map TODO:
- [ ] Revise topics (add/remove/rename)
- [ ] Add topic text (or disable expansion box for topics without it)
- [ ] Resize it on various screen widths (e.g. too big on mobile currently)
- [ ] Review design/style of projects on the topic map (or remove)
- [ ] Add remaining projects and link them

Other TODO:
- [ ] Finish adding and linking projects in the "Our Work" section

![topicmap](https://github.com/life-itself/community/assets/960919/10f2159b-3f1b-4a36-8a94-4976d69dd822)

Preview: https://a477eca2.lifeitself-org.pages.dev/research

rufuspollock commented:
@mattosborn i think this is good to go for now and we can add more stuff later. Are you happy for me to merge?

mattosborn commented:
> @mattosborn i think this is good to go for now and we can add more stuff later. Are you happy for me to merge?

I added the missing links, fixed some things and disabled everything that was unfinished so feel free to merge 👍
